src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclint/Checker.java

```diff
@@ -985,7 +985,7 @@ public Void visitReturn(ReturnTree tree, Void ignore) {
         }
         if (tree.isInline()) {
             DocCommentTree dct = getCurrentPath().getDocComment();
-            if (tree != dct.getFirstSentence().get(0)) {
+            if (dct.getFullBody().isEmpty() || tree != dct.getFullBody().get(0)) {
```

Comment:
It should not be necessary to resort to `getFullBody`.  It should be enough to check the first sentence, but that check should not throw an exception.

Reply:
I thought that getting the first sentence would unnecessarily trigger sentence segmentation and read less clearly. But I can revert it if you like.

Reply:
The split happens when the tree is created. See `DocTreeMaker` line 198.